Added exercise for `Armstrong numbers`

suraj-mandal opened the pull request:
Hi,

Previously I had created a pull request for `reverse-string`. However that was very easy so chances of it getting accepted are low. As a result, I have created another exercise called `armstrong-numbers`.

### Note:

This PR contains the files for the folder `reverse-string`. Do you want me to remove them? and update the config.json file so that it is not available.

jiegillet commented:
Hi! Haha nice, another one. 
yes please remove all the reverse string files. I will wait until I have access to a computer before reviewing this one, but pretty much all comments I left for your other PR are applicable to this one. 

Also, just in case you plan on doing more (which would be great) can you avoid perfect-numbers? I’m planning on building it on the Exercism twitch stream Thursday.

suraj-mandal commented:
Sure.. i will not touch perfect numbers then

suraj-mandal commented:
I have removed the reverse-string folder and updated config.json for the armstrong-numbers with the required values.

lpil commented on config.json:
```diff
@@ -16,7 +16,7 @@
   },
   "test_runner": {
-    "average_run_time": 3.0
+    "average_run_time": 3
```
I think this line may want to be reverted

lpil commented on config.json:
```diff
@@ -562,6 +562,21 @@
+        "prerequisites": [
+            "list",
+            "bool",
+            "int"
```
Could we use 2 space indentation here? Thank you

suraj-mandal replied:
Sure. Making the changes here

lpil reviewed (commented):
Very nice! Thank you. The solution looks great to me

suraj-mandal commented:
I have updated the code. Made the following changes

1. Ran an overall gleam format along with check to make sure that the example code is properly formatted.
2. Modified the config.json file to make the average_test_runner value to 3.0 instead of 3
3. Added 2-space indentation
4. Added author name in the ./.meta/config.json

lpil reviewed (approved):
Thank you!
